backend/app/views/spree/admin/orders/_shipment.html.erb

```diff
@@ -1,7 +1,11 @@
 <% admin_breadcrumb(plural_resource_name(Spree::Shipment)) %>
 
 
-<% manifest_items = Spree::ShippingManifest.new(inventory_units: shipment.inventory_units.where(carton_id: nil)).items %>
+<%
+  manifest_items = Spree::ShippingManifest.new(
+    inventory_units: shipment.inventory_units.where(carton_id: nil),
+  ).items.sort_by { |item| item.line_item.created_at }
```

Comment:
I like how you chose to sort this by when the line item was created. Exactly how I would expect it to be sorted. 👍